### PR TITLE
Ddcnl 3945

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: app/views/session_expired.scala.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@*
```
Comment (struong) [inline app/views/session_expired.scala.html:1]: Rename to SessionExpiredView

File: app/views/unauthenticated_main.scala.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+@(title: String,
+  sidebarLinks: Option[Html] = None,
+  sidebarClass: Option[String] = None,
+  articleClasses: Option[String] = Some("full-width"),
+  includeGridWrapper: Boolean = false,
+  script: Option[Html] = None,
+  pageHeader: Option[Html] = None,
+  displayTimeoutDialog: Boolean = true
```
Comment (struong) [inline app/views/unauthenticated_main.scala.html:46]: Is this used in this file?

File: app/views/main.scala.html [hunk 3]
```diff
@@ -50,7 +50,8 @@
   optimizelyAudience: Option[String] = None,
   pageHeader: Option[Html] = None,
   script: Option[Html] = None,
-  hidePtaAccountNav: Boolean = false
+  hidePtaAccountNav: Boolean = false,
+  displayTimeoutDialog: Boolean = true
```
Comment (struong) [inline app/views/main.scala.html:54]: Can't see where this is being set too false, do you need it?

File: app/views/session_expired.scala.html (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/session_expired.scala.html]: done

File: app/views/main.scala.html (hunk 3 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/main.scala.html]: yes its not used as only unauthenticated main is used for session expired page where we dont want to display timeout dialog so i've removed it

File: app/views/unauthenticated_main.scala.html (hunk 2 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/unauthenticated_main.scala.html]: removed

File: app/uk/gov/hmrc/tai/config/ApplicationConfig.scala [hunk 4]
```diff
@@ -109,6 +109,9 @@ class ApplicationConfig @Inject()(
 
   lazy val citizenAuthFrontendSignOutUrl: String = citizenAuthHost + "/ida/signout"
 
+  lazy val sessionTimeoutInSeconds: Int = 900
+  lazy val sessionCountdownInSeconds: Int = 120
```
Comment (struong) [inline app/uk/gov/hmrc/tai/config/ApplicationConfig.scala:113]: Can we read these from a config? We sometimes want to change it on different environments to help testing

Comment (Muhammad-Ali-Ahmad) [reply app/uk/gov/hmrc/tai/config/ApplicationConfig.scala]: done

**Pull request merged**